src/storage/src/hummock/store/local_hummock_storage.rs

```diff
@@ -106,34 +106,36 @@ impl LocalHummockFlushedSnapshotReader {
         hummock_version_reader: &HummockVersionReader,
         read_version: &HummockReadVersionRef,
         table_key: TableKey<Bytes>,
-        epoch: u64,
         read_options: ReadOptions,
     ) -> StorageResult<Option<StateStoreKeyedRow>> {
         let table_key_range = (
             Bound::Included(table_key.clone()),
             Bound::Included(table_key.clone()),
         );
 
-        let (table_key_range, read_snapshot) =
-            read_filter_for_version(epoch, read_options.table_id, table_key_range, read_version)?;
+        let (table_key_range, read_snapshot) = read_filter_for_version(
+            MAX_EPOCH,
+            read_options.table_id,
+            table_key_range,
+            read_version,
+        )?;
 
         if is_empty_key_range(&table_key_range) {
             return Ok(None);
         }
 
         hummock_version_reader
-            .get(table_key, epoch, read_options, read_snapshot)
+            .get(table_key, MAX_EPOCH, read_options, read_snapshot)
```

Comment:
Why the `MAX_EPOCH` is used here and below instead of the `epoch`.

Reply:
The semantic of `LocalHummockFlushedSnapshotReader` is to read the latest uncommitted data, therefore we use `MAX_EPOCH` to specify reading the latest data.